app/bundles/CampaignBundle/Entity/Event.php

```diff
@@ -654,44 +650,42 @@ public function addChild(Event $children)
 
     /**
      * Remove children.
-     *
-     * @param \Mautic\CampaignBundle\Entity\Event $children
      */
     public function removeChild(Event $children)
     {
         $this->children->removeElement($children);
     }
 
     /**
-     * @return ArrayCollection|Event[]
+     * @return \Doctrine\Common\Collections\Collection<int, \Mautic\CampaignBundle\Entity\Event>|array<int, \Mautic\CampaignBundle\Entity\Event>
      */
-    public function getChildren()
+    public function getChildren(): Collection|array
```

Comment:
The `children` property can only hold a `Collection` value. Is the return type of `array` valid?